tests/chainer_tests/links_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -258,8 +259,34 @@ def setUp(self):
                                              initial_beta=self.initial_beta)
 
     @condition.retry(3)
-    def test_initialize(self):
-        self.assertTrue((self.initial_gamma == self.link.gamma.data).all())
-        self.assertTrue((self.initial_beta == self.link.beta.data).all())
+    def test_initialize_cpu(self):
+        assert_allclose(self.initial_gamma, self.link.gamma.data)
```

Comment:
Use `testing.assert_allclose` because OpenStack Style Guidelines, which we follow, forbids to import objects (see [here](http://docs.openstack.org/developer/hacking/#imports))